src/main/resources/org/jenkinsci/plugins/ownership/model/workflow/OwnershipGlobalVariable/help.jelly

```diff
@@ -0,0 +1,69 @@
+<?jelly escape-by-default='true'?>
+<div>
+ Provides ownership info for different item types.
+ <p/>
+ Currently the macro supports retrieval of ownership info for jobs and nodes only.
+ It can be done via the nested variables:
+<dl>
+  <dt>ownership.job</dt>
+  <dd>
+    Job ownership info.
+    This info may be inherited from parent folders.
+  </dd>
+  <dt>ownership.node</dt>
+  <dd>
+    Node ownership info. 
+    The expression must be used within the <code>node()</code> Pipeline block.
+    Otherwise the command will throw an exception.
+   </dd>
+ </dl>
+ 
+ Each call returns a structure, which provides the following fields:
+ <dl>
+   <dt>ownershipEnabled</dt>
```

Comment:
Just to keep compatibility with the legacy API of OwnersgipDescription. Subject for refactoring